test/e2e/operator_test.go

```diff
@@ -757,6 +758,76 @@ func TestHostNetworkEndpointPublishingStrategy(t *testing.T) {
 	}
 }
 
+// TestHostNetworkPortBinding creates two ingresscontrollers on the same node
+// with different port bindings and verifies that both routers are available.
+func TestHostNetworkPortBinding(t *testing.T) {
+	// deploy first ingresscontroller with the default port bindings
+	name1 := types.NamespacedName{Namespace: operatorNamespace, Name: "host"}
+	ing1 := newHostNetworkController(name1, name1.Name+"."+dnsConfig.Spec.BaseDomain, nil, nil)
+	if err := kclient.Create(context.TODO(), ing1); err != nil {
+		t.Fatalf("failed to create the first ingresscontroller: %v", err)
+	}
+	defer assertIngressControllerDeleted(t, kclient, ing1)
+
+	conditions := []operatorv1.OperatorCondition{
+		{Type: ingresscontroller.IngressControllerAdmittedConditionType, Status: operatorv1.ConditionTrue},
+		{Type: operatorv1.IngressControllerAvailableConditionType, Status: operatorv1.ConditionTrue},
+		{Type: "DeploymentReplicasAllAvailable", Status: operatorv1.ConditionTrue},
+		{Type: operatorv1.LoadBalancerManagedIngressConditionType, Status: operatorv1.ConditionFalse},
+		{Type: operatorv1.DNSManagedIngressConditionType, Status: operatorv1.ConditionFalse},
+	}
```

Comment:
Is there anything special about this list of conditions, or is this a general set of conditions that should be expected on an ingresscontroller with `spec.endpointPublishingStrategy.type: HostNetwork`?  If the latter, can you use a named variable à la `availableConditionsForIngressControllerWithNodePort` etc.?